### ISSUE TITLE
HOW TO PARSE COUNTRY NAME IN PROXY CHECKER?

### ISSUE THREAD

**Issue by Samsung2023** [mohamm4dx/SilverBullet#86]
I'm searching a way to parse proxy country in proxy checker, is there a way? 
this is the response of the proxy URL I'm using:
`{"country":"Germany","regionName":"Hesse","city":"Frankfurt am Main","query":"45.87.XXX.XX"}`

**Comment by RifHut** [comment]
use json path parsing option !$.country

**Comment by Samsung2023** [comment]
> @Samsung2023
> 
> USE LR mode
> 
> its gonna something like this Just copy paste the code into your SVB loliscript and switch back to STACK VIEW
> 
> PARSE "" LR ""query":"" """ -> VAR "IP"
> 
> PARSE "" LR ""port":"" """ -> VAR "PORT"
> 
> FUNCTION Constant ":" -> CAP "Proxy"



No, I want to show the proxies's country in proxies database in silverbullet. you can't do LR there.
